literature.bib

```diff
@@ -3386,22 +3386,22 @@ @article{RTS+:JLAMP16
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2015 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 @inproceedings{PyEDA15,
-	author 		= {Chris Drake},
-	title  		= {{PyEDA: Data Structures and Algorithms for Electronic Design Automation}},
-	booktitle 	= {{Proc.\ Conf.\ on Python in Science (SciPy)}},
-  	pages     	= {25--30},
-	editor  	= {Kathryn Huff and James Bergstra},
-	year   		= 2015,
+	author = {Chris Drake},
+	title = {{PyEDA: Data Structures and Algorithms for Electronic Design Automation}},
+	booktitle = {{Proc.\ Conf.\ on Python in Science (SciPy)}},
```

Comment:
Is there a reason why this is not in MYabrv and MYshort?

Reply:
As before, single occurrence, will move to MYabrv/MYshort once a second occurrence is added.